src/Common/RegisterPage.js

```diff
@@ -0,0 +1,47 @@
+import { StyleSheet } from 'react-native';
```

Comment:
Pick a different filename to better indicate this file contains styling content. And, can we place it in Personal folder, since those stylings are not reused in both stats and creation pages